### PR TITLE
Create install script to install dependencies

### PR DESCRIPTION
**Pull request by kinsaurralde**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
+sudo apt update && sudo apt install terraform
+
+# Install kubectl.
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+curl -LO "https://dl.k8s.io/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
+echo "$(cat kubectl.sha256)  kubectl" | sha256sum --check
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+kubectl version --client
+
```
Comment (manitgupta) [inline install.sh:19]: Can we also add maven here?

**Review by manitgupta — CHANGES_REQUESTED** [review]
We should have Maven setup as part of this script.
If we can have JMeter setup as well that would be an awesome bonus.

File: install.sh (hunk 1 above)
Comment (kinsaurralde) [reply install.sh:19]: Added maven. Im not sure about JMeter, I dont remember installing it.

Comment (manitgupta) [reply install.sh:19]: I actually meant installing `mvn` and not building the project.
Building the project can live outside of the install script. Installation of maven can be part of it.

Ref -https://maven.apache.org/install.html

**Review by manitgupta — APPROVED** [review]

**Review by manitgupta — APPROVED** [review]

**Pull request merged**